assets/js/blocks/classic-template/editor.scss

```diff
@@ -51,7 +52,15 @@
 	align-items: center;
 	gap: $gap-small;
 
+	@media only screen and (max-width: 480px) {
+		gap: $gap-smaller;
+	}
+
 	span {
 		@include font-size(larger);
+
+		@media only screen and (max-width: 480px) {
+			@include font-size(regular, 1.25rem);
+		}
```

Comment:
Are you sure that it is necessary? I think that we can remove it.

Reply:
On mobile screens, the font size is huge. This media query makes it appropriate for mobile screen sizes.

Reply:
Yes I am sure. On mobile screens, the font size is huge. It wraps it to the next line and looks odd.
This media query makes it appropriate for mobile screen sizes, especially for the ones with lower resolutions.